stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -176,18 +176,20 @@ class AssignManualQuantsLines(models.TransientModel):
 
     @api.onchange("selected")
     def _onchange_selected(self):
```

Comment:
@xavier-bouquiaux Why not changing the field qty to computed/stored/readonly=False field to avoid pre-deprecated onchange methods ?

Reply:
Hi Denis

The goal here is to make the module bug free (changes in the view) and i also made some changes in code for obvious changes

But i don't have time for this kind a refactor at the moment

may-be create an issue and we'll see afterwards ?